packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -861,13 +866,12 @@ class _TableState<T> extends State<_Table<T>> with AutoDisposeMixin {
       setState(() {
         final Selection<T> selection = oldWidget.selectionNotifier!.value;
         if (selection.scrollIntoView) {
-          final int selectedDisplayRow = (selection.node! as dynamic).nodeIndex;
+          final int selectedDisplayRow = selection.nodeIndex!;
           // TODO(terry): Optimize selecting row, if row's visible in
           //              the viewport just select otherwise jumpTo row.
           final newPos = selectedDisplayRow * defaultRowHeight;
 
-          // TODO(terry): Should animate factor out _moveSelection to reuse here.
-          scrollController.jumpTo(newPos);
+          scrollToPosition(scrollController, newPos);
```

Comment:
I'm fairly certain all other changes to table.dart outside of these two lines can be reverted to their previous state. Let's do that.